### ISSUE TITLE
Usage of call_genotype with no sex gene

### ISSUE THREAD

**Issue by piosierra** [shohei-kojima/MEGAnE#1]
Hello,

I am trying to run the pipeline for a species with no sex gene (or not determined so far). Is there a way to by pass the requirement for a name for -female_sex_chr?

Thanks

**Comment by shohei-kojima** [comment]
Thank you for your feedback.

From v1.0.1beta, MEGAnE has `-no_sex_chr` option. Please pull the new version from [DockerHub](https://hub.docker.com/r/shoheikojima/megane) and use that option. With this option, MEGAnE treats all chromosomes as autosomes. An example usage of this option is available in the [wiki page](https://github.com/shohei-kojima/MEGAnE/wiki/Run-with-a-custom-repeat-library-and-annotation-(mostly-for-analysis-of-non-human-WGS)).

**Comment by piosierra** [comment]
I am using the singularity version. Should it update all the same? (I just downloaded it today)

**Comment by shohei-kojima** [comment]
Yes, a singularity container and a docker container built from the DockerHub will be the same. Did you specify the tag? If not, please try the build again.

```
sudo singularity build MEGAnE.sif docker://shoheikojima/megane:v1.0.1.beta
# or
singularity build --fakeroot MEGAnE.sif docker://shoheikojima/megane:v1.0.1.beta
```

**Comment by piosierra** [comment]
Updated. Thanks!
